src/gui/EditWidgetIcons.cpp

```diff
@@ -223,8 +223,15 @@ void EditWidgetIcons::fetchFavicon(const QUrl& url)
     }
 
     m_httpClient->setConnectingTimeOut(5000, [this]() {
-        resetFaviconDownload();
-        MessageBox::warning(this, tr("Error"), tr("Unable to fetch favicon."));
+        if (m_url.startsWith("http://")) {
+            resetFaviconDownload();
+            MessageBox::warning(this, tr("Error"), tr("Unable to fetch favicon."));
+        } else {
+            m_url = m_url.replace("https","http");
```

Comment:
@TheZ3ro don't know if it would be an ok workaround but if a fully qualified URL can't be found it would fall back into pulling a string that looks like a consistent domain name from the end of the string?

Quite many "fallbacks" required to support all kinds of scenarios.